index.html

```diff
@@ -0,0 +1,149 @@
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/exhibition.png" class="program-image-icons" alt="illustration showing an exhibition">
+            <div class="program-description-title">
+              <p>CC Exhibition </p>
+            </div>
+            <div class="program-description-text">
+              Appreciate various creators showing off their locally brewed art pieces.
+            </div>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/forum.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Forum
+            </p>
+            <p class="program-description-text">
+              Have the time to share your thoughts and opinions with experts for each topic.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/workshop.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Workshop
+            </p>
+            <p class="program-description-text">
+              Try creating your own creative/ technological work in groups based on your interests.
+            </p>
+          </div>
+          <div class="grid-program-container">
+              <img src="./images/ignite.png" class="program-image-icons" alt="illustration showing a discussion">
+            <p class="program-description-title">
+              Nkwashi Ignite
+            </p>
+            <p class="program-description-text">
+              Get opportunities to network with atendees, affliliates from around the world and the presenters/ authors.
+            </p>
+          </div>
+        </div>
+        <a id="join-conference" class="join-button p-2" href="#" role="button"> JOIN THE NKWASHI CONFERENCE </a>
+      </div>
+    </section>
+
+    <section class="speakers">
+      <h2 class="text-center speaker-title"> Featured Speakers</h2>
+      <hr class="orange-dividing-line-about mx-auto">
+      <div id="speakers" class="grid-speaker-container">
+      </div>
+      <button id="see-less-button" class="see-less-button-hide speakers-button rounded p-2" type="button"> SEE LESS <i class="fas fa-chevron-down see-more-speakers-icon>"></i> </button>
+    </section>
+
+    <section class="mobile-hidden partners text-center py-3">
+      <h2> Partners </h2>
+      <hr class="orange-dividing-line-about mx-auto">
+      <div class="partner-image-grid m-4">
+          <img class="partner-logo-airbnb" alt="airbnb logo" src="./images/Airbnb-logo.png">
+          <img class="partner-logo-firefox" alt="firefox logo" src="./images/firefoxLogo.png">
+          <img class="partner-logo-google" alt="google logo" src="./images/Google-logo.png">
+      </div>
+    </section>
```

Comment:
- [ ] The partners section is hidden on mobile screen kindly add it.
**Screenshot** 👇
<img src='https://user-images.githubusercontent.com/33323077/125158479-60654f80-e169-11eb-8fca-c2620b63ddee.png' width='200px'/>